File: 트랜스폼컨쿼/LCM.py
def GCD(a,b):
    while b!=0:
        a,b=b,a%b
    return a
def LCM(a,b):
    result=int((a*b)/GCD(a,b))
    return result
string=input("LCM을 구할 두 수를 입력하세요 : ").split()
print("%d와 %d의 LCM : %d"%(int(string[0]),int(string[1]),LCM(int(string[0]),int(string[1]))))

File: 트랜스폼컨쿼/LCM.c
#include <stdio.h>
int GCD(int,int);
int LCM(int,int);
int main(void){
    int x,y;
    printf("LCM을 구할 두 수를 입력하세요 : ");
    scanf("%d %d",&x,&y);
    printf("%d와 %d의 LCM : %d\n",x,y,LCM(x,y));

    return 0;
}
int GCD(int a, int b){
    int temp;
    while(b){
        temp=b;
        b=a%b;
        a=temp;
    }
    return a;
}
int LCM(int a, int b){
    int result=(a*b)/GCD(a,b);
    return result;
}